server.js

```diff
@@ -1,19 +1,56 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import topMusicData from "./data/top-music.json";
+const listEndpoints = require('express-list-endpoints')
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+//Should I use docker for below issue or what do I need to do to solve it?
+//I couldn't make the MONGO_URL work with the localhost-version - when I did that it stopped working locally
```

Comment:
The way I understand it, the MONGO_URL is for deployment, whereas "mongodb://localhost/project-mongo" is for what we can use, to see our work in Compass as we're developing it. So no need to change the link that was already there (álthought for some of us, "mongodb://127.0.0.1:27017/project-mongo" works better, for some reason)

Reply:
Thanks Emma - Yeah I noticed that too :D